src/ZProbe.cpp

```diff
@@ -24,6 +24,7 @@ void ZProbe::Init(float h)
 	tolerance = DefaultZProbeTolerance;
 	maxTaps = DefaultZProbeTaps;
 	invertReading = turnHeatersOff = false;
+	eEndstop = 1; // Default e1 for Mode 6 backward compatibility
```

Comment:
I just realized that this would "break" default Mode 4 behavior.
I have no quick fix for a nicer way to handle that right now.